docs/specs/validation.yml

```
@@ -1568,3 +1568,144 @@ outputs:
     {"message_severity":"suggestion","log_item_type":"user","code":"duplicate-h1","message":"H1 'h1' is duplicated with other articles: 'docs/v1/b.md, docs/v1/token1.md, docs/v1/token2.md(5,1)'","file":"docs/v1/token2.md","line":5,"end_line":5,"column":1,"end_column":1}
   2420f88b/docs/a.json:
   2420f88b/docs/b.json:
+---
+#TOC validation - toc-deprecated
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "toc": {
+        "name": "TOC and Breadcrumbs ",
+        "description": "Validates TOC and breadcrumbs.",
+        "rules": [
+          {
+            "type": "TocDeprecated",
+            "message": "Markdown TOC files are deprecated. Convert to a YAML TOC.",
+            "code": "toc-deprecated",
+            "severity": "SUGGESTION",
+            "exclusions": ["includes", "landingpage", "hubpage", "conceptual"]
+          }
+        ]
+      }
+    }
+  a/toc.md:
```

The linux test failed because it is expecting an upper case `TOC.md`.

```suggestion
  a/TOC.md:
```